hat/assets/js/apps/Iaso/domains/projects/components/FeatureFlagsSwitches.tsx

```diff
@@ -24,52 +27,164 @@ const styles = theme => ({
 
 type Props = {
     featureFlags: FeatureFlag[];
-    projectFeatureFlagsValues: (string | number)[];
-    handleChange: (newValue: any) => void;
-    isLoading;
+    projectFeatureFlags: ProjectFeatureFlag[];
+    onFeatureFlagsChanged: (newValue: ProjectFeatureFlag[]) => void;
+    isLoading: boolean;
 };
 
 const useStyles = makeStyles(styles);
+
+const featureFlagToProjectFeatureFlag = (
+    ff: FeatureFlag,
+): ProjectFeatureFlag => {
+    const configuration: Record<string, any> = {};
+    if (ff.configuration_schema != null) {
+        Object.entries(ff.configuration_schema).forEach(([k, conf]) => {
+            configuration[k] = conf.default;
+        });
+    }
+    return {
+        ...ff,
+        configuration,
+    };
+};
+
+const translateType = (type: string): InputComponentType => {
+    switch (type.toLowerCase()) {
+        case 'int':
+        case 'long':
+        case 'double':
+        case 'float':
+        case 'decimal':
+            return 'number';
+        case 'url':
+        case 'str':
+        case 'string':
+            return 'text';
+        default:
+            return 'text';
+    }
+};
+
+export const useFeatureFlagConfiguration = (
+    ff: FeatureFlag,
+    onFeatureFlagChanged: (changed: ProjectFeatureFlag) => void,
+    pff?: ProjectFeatureFlag,
+) => {
+    if (ff.configuration_schema == null) {
```

Comment:
In this case, ff.configuration_schema == null would work the same as ff.configuration_schema === null because null is a primitive value, but using === is the preferred practice. This is why eslint is not complaining here 
```suggestion
    if (ff.configuration_schema === null) {
```